pkg/connector/resource_types.go

```diff
@@ -30,4 +30,10 @@ var (
 		Description: "Dataset of Google BigQuery",
 		Traits:      []v2.ResourceType_Trait{v2.ResourceType_TRAIT_APP},
```

Comment:
I'd remove the app trait from service accounts, datasets, and projects. It's not a trait that makes sense for this connector.

Reply:
Fixed